lib/services/networkManagement2/lib/models/expressRouteCircuitRoutesTableSummary.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ * 
+ * Code generated by Microsoft (R) AutoRest Code Generator 0.16.0.0
+ * Changes may cause incorrect behavior and will be lost if the code is
+ * regenerated.
+ */
+
+'use strict';
+
+/**
+ * @class
+ * Initializes a new instance of the ExpressRouteCircuitRoutesTableSummary class.
+ * @constructor
+ * The routes table associated with the ExpressRouteCircuit
+ * @member {string} [neighbor] Neighbor.
+ * 
+ * @member {number} [v] V.
+ * 
+ * @member {number} [as] AS.
+ * 
+ * @member {string} [upDown] upDown.
+ * 
+ * @member {string} [statePfxRcd] statePfxRcd .
+ * 
```

Comment:
@deepakswifty - I think these properties should be better described in the swagger spec. As of now the end user cannot make any sense of this.